service/service.go

```diff
@@ -78,6 +78,7 @@ func GetRouter(ctx context.Context, as *APIServer, uis *UIServer) (http.Handler,
 	// endpoints.
```

Comment:
Can we add a comment that `OPENAPI_VERSION` is a placeholder that's replaced with the proper version?

Reply:
I added an explanation above explaining the placeholder